python/chronos/test/bigdl/chronos/forecaster/test_lstm_forecaster.py

```diff
@@ -542,7 +542,7 @@ def test_predict_interval(self):
                                     lr=0.01)
         forecaster.fit(train_data, epochs=2)
         y_pred, std = forecaster.predict_interval(data=test_data[0],
-                                                  val_data=val_data,
+                                                  validation_data=val_data,
                                                   repetition_times=5)
         assert y_pred.shape == test_data[1].shape
         assert y_pred.shape == std.shape
```

Comment:
should we test `predict_with_openvino` and loader/tsdataset here?

Reply:
will add a ut for this.